libs/seiscomp/gui/datamodel/eventlistview.cpp

```diff
@@ -1027,8 +1045,13 @@ class FocalMechanismTreeItem : public SchemeTreeItem {
 				setText(config.columnMap[COL_REGION], Regions::getRegionName(lat, lon).c_str()); // Region
 			}
 
+			// add publication marker
 			char stat = objectStatusToChar(fm);
-			setText(config.columnMap[COL_TYPE], QString("%1").arg(stat));
+			setText(config.columnMap[COL_TYPE],
+					QString("%1%2")
```

Comment:
Either mix tabs and spaces here to align this and all subsequent lines with `config.columnMap`.

Reply:
Or add a new line right after `setText(`:

```c++
setText(
<tab>a,
<tab>b,
<tab>c
);
```